docs/edgeql/funcops/datetime.rst

```diff
@@ -67,10 +70,22 @@ Date and Time
 
 
 .. eql:operator:: DTPLUS: datetime + duration -> datetime
+                          datetime + cal::relative_duration \
```

Comment:
I'd keep the `duration` group as is, and add the `relative_durations` ops after it. I.e. don't interleave them like that, it will be hard to read/follow.